packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -378,25 +389,34 @@ export class FetchInstrumentation extends InstrumentationBase<
         ) {
           try {
             endSpanOnError(span, error);
+          } catch(e) {
```

Comment:
Is this actually an instrumentation error? I think this is instrumenting an error in the library.

Reply:
I am not sure I understand. `endSpanOnError` is OTEL code, OTEL is used for instrumentation. If it throws then it is an error with the instrumentation no?